deploy/docs/V1_Migration_doc.md

```diff
@@ -0,0 +1,156 @@
+# Kubernetes Collection 1.0.0 - Breaking Changes
+
+<!-- TOC -->
+
+- [Helm Users](#helm-users) 
+  - [Changes](#changes) 
+  - [How to upgrade](#how-to-upgrade) 
+    - [1. Upgrade to helm chart version `v0.17.1 `](#1-upgrade-to-helm-chart-version-v0.17.1)
+    - [2: Run upgrade script](#2-run-upgrade-script)
+- [Non-Helm Users](#non-helm-users) 
+  - [Changes](#breaking-changes) 
+  - [How to upgrade](#how-to-upgrade-for-non-helm-users)
+    - [1. Move environment variable config changes to Fluentd pipeline](#1-move-environment-variable-config-changes-to-fluentd-pipeline)
+    - [2. Modify filter based on kubernetes version](#2-modify-filter-based-on-kubernetes-version))
+    - [3: Apply new changes](#3-apply-new-changes)
+
+
+<!-- /TOC -->
+
+Based on the feedback from our users, we will be introducing several changes to the Sumo Logic Kubernetes Collection solution. Here we detail the changes for both Helm and Non-Helm users, as well as the exact steps for migration.
+## Helm Users
+### Changes
+
+- Falco installation disabled by Default
```

Comment:
We also bumped the helm chart version which included a fix to disable the botcoin miner rule by default - https://github.com/falcosecurity/falco/issues/835

Reply:
Can we add a note on how to enable.  also does the migration script respect the setting from the users previous values.yaml? E.g. if they had it enabled in the values.yaml, will it respect that?

Reply:
https://github.com/helm/charts/blob/master/stable/falco/rules/falco_rules.yaml#L2652 @frankreno how will we enable this? In the `values.yaml` there is only an option to just specify the rules file path.

Reply:
On the enable, i just mean that when a user upgrades, they previously had falco.enabled=true.  Now they have falco.enabled=false.  In upgrade path, we should keep it enabled, not disable.  It should only disabled for new installs as if upgrading and it was enabled it is working most likely.  If the user set falco.enabled=false, we should keep it disabled.  Sorry I was not very clear the first time.That is what this comment was about.

Reply:
ok got it, thanks. cc @samjsong to add this check in the upgrade script.